pkgs/development/python-modules/csv-diff/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pytest
+, pytest-runner
+, dictdiffer
+, click
+}:
+
+buildPythonPackage rec {
+  pname = "csv-diff";
+  version = "1.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1skbaxgn72qlpl8pmmmihpzlb28rvks78wwi9yydhzf6j9wi357z";
+  };
+
+  propagatedBuildInputs = [
+    dictdiffer
+    click
+  ];
+
+  checkInputs = [
+    pytest
+    pytest-runner
```

Comment:
```suggestion
```
and remove the requirement from https://github.com/simonw/csv-diff/blob/1.1/setup.py#L26

Reply:
I can do that, but why?

Reply:
It can't work because we have not network connection and can only cause issues.